docs/get-started.asciidoc

```diff
@@ -0,0 +1,265 @@
+If you need more control over your configuration after getting set up, you can learn more in the https://opentelemetry.io/docs/languages/net[OpenTelemetry SDK documentation].
+
+[NOTE]
+====
+As an OpenTelemetry SDK, the distro supports sending data to any OpenTelemetry protocol (OTLP) endpoint (https://opentelemetry.io/docs/collector/[OpenTelemetry Collector]), but this guide assumes you are sending data to Elastic.
+====
+
+////
+✅ What the user needs to know and/or do before they install the distro
+////
+[discrete]
+[[get-started-prerequisites]]
+=== Prerequisites
+
+Before getting started:
+
+* *Check your .NET SDK version.* The current documentation and examples are written with .NET 6 and newer applications in mind. Before continuing, ensure that you have a supported https://dotnet.microsoft.com/en-us/download/dotnet[.NET SDK version] installed locally.
++
+* *Set up Elastic Observability*. You'll need somewhere to send the gathered OpenTelemetry data, so it can be viewed and analyzed. This doc assumes you're using an https://www.elastic.co/observability[Elastic Observability] cloud deployment. You can use an existing one or set up a new one.
++
+[[elastic-observability-setup]]
+[%collapsible]
+.*Expand for setup instructions*
+====
+This section shows how to create an https://www.elastic.co/observability[Elastic Observability] cloud deployment and get the data you need to configure the distro to send data to it.
+
+. Register at https://cloud.elastic.co/registration[cloud.elastic.co], if you haven't already. This supports starting **free trial**.
+. After registering, click *Create deployment* at `<https://cloud.elastic.co/home>`.
+. Once that is created, click *Open* to visit your {kib} home page, (for example, `https://{DEPLOYMENT_NAME}.kb.{REGION}.cloud.es.io/app/home#/getting_started`).
+====
+
+////
+✅ How to install the distro
+////
+[discrete]
+[[get-started-install]]
+=== Install
+
+////
+✅ Step-by-step instructions
+////
+To get started with the Elastic OpenTelemetry Distribution for .NET, add the `Elastic.OpenTelemetry` NuGet package reference to your project file:
+
+[source,xml]
+----
+<PackageReference Include="Elastic.OpenTelemetry" Version="0.1.0-alpha.1" />
+----
+
+After adding the package reference, you can start using the distro
+in your application.
+
+////
+✅ Any additional info related to installation
+////
+[NOTE]
+.The Elastic distro and the OpenTelemetry SDK
+====
+The distro includes a transitive dependency on the OpenTelemetry SDK, so you do not _need_ to add the
+OpenTelemetry SDK package to your project directly. However, you _can_ explicitly add the OpenTelemetry SDK as a dependency if you want to opt into newer SDK versions. If you do this, the OpenTelemetry SDK dependency must
+be defined _before_ the Elastic OpenTelemetry Distribution for .NET is defined.
+====
+
+[NOTE]
+.Migrating from the OpenTelemetry SDK to the Elastic distro
```

Comment:
I'm curious about this. On one hand, I see why it fits here. On the other, I can see it being its own section or own page. I guess it's a really small update if someone is migrating which is why you've added it in the way you did.

Reply:
Not sure what I'm trying to say. Likely fine as-is here.

Reply:
Yea. The Java docs has a full page for migrating from the APM agent to the Elastic distro, these docs have this note about going from the Otel SDK to the Elastic distro, and the Node docs have neither. At least for 8.15, I've been [considering migration content as optional](https://docs.google.com/document/d/1W0pvuEdVQpa8_dpiBBO0mQwZXZwTgBu8S4SkV40kjJQ/edit#heading=h.afljcowgd5on) (i.e. not a part of "minimal viable docs"), but I'm open to other opinions.

Reply:
No, I think that's the right call. If the experience is different then there's no need for us to standardize the approach we take to documenting it.